packages/field/src/useField.ts

```diff
@@ -9,12 +9,12 @@ import { useState } from 'react';
 import { generateId } from '@zendeskgarden/container-utilities';
 
 export interface IUseFieldPropGetters {
-  getHintProps: <T>(options?: T & React.HTMLProps<any>) => React.HTMLProps<any>;
-  getLabelProps: <T>(options?: T & React.HTMLProps<any>) => React.HTMLProps<any>;
+  getHintProps: <T>(options?: T) => T & React.HTMLProps<any>;
+  getLabelProps: <T>(options?: T) => T & React.HTMLProps<any>;
   getInputProps: <T>(
     options?: T & React.HTMLProps<any>,
```

Comment:
This could just be `T`, right?

Reply:
Oh yes, updated. Thanks for catching this!